### PR TITLE
Feature/header sidebar

### PR DESCRIPTION
**Pull request by mctolentino**
This PR is to resolve issue #6

### CODE REVIEW

File: src/containers/sidebar.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import Drawer from "@material-ui/core/Drawer";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
+import IconButton from "@material-ui/core/IconButton";
+import ChevronLeftIcon from "@material-ui/icons/ChevronLeft";
+import InfoIcon from "@material-ui/icons/Info";
+import SchoolIcon from "@material-ui/icons/School";
+import AccountIcon from "@material-ui/icons/AccountCircle";
```
Comment (denibulkashvili) [inline src/containers/sidebar.js:12]: Giving me import error. Should it be `import AccountCircle`?

Comment (mctolentino) [reply src/containers/sidebar.js:12]: Screenshot from 2019-03-28 19-07-58

Doesn't seem to show any errors on my part, I'm just explicitly naming the default export so it's clear that its an icon.

Comment (denibulkashvili) [reply src/containers/sidebar.js:12]: yep, all works fine. 

also here's a nitpick, you could reduce all your imports to 
```{Divider, Drawer, IconButton, List, ListItem, ListItemIcon, ListItemText, Typography} from "@material-ui/core";```
Great job on that sidebar component btw

**Review by denibulkashvili — APPROVED** [review]

File: src/containers/sidebar.js (hunk 1 above)
Comment (mctolentino) [reply src/containers/sidebar.js:12]: Thanks! I was planning to declare them that way but in the documentation of Material UI, it says that we should use that way of declaring if the environment supports "tree-shaking". I was not sure if our build system supports it so opted for the more verbose approach.

https://material-ui.com/style/icons/ (Imports section)

Comment (denibulkashvili) [reply src/containers/sidebar.js:12]: Oh, I see. I've never heard of *tree shaking* before. It looks like a way of instructing webpack to eliminate any unused imports, and with our config, we're not doing that. But we can leave imports as they are just be safe

Comment (davidkwan95) [reply src/containers/sidebar.js:12]: @denibulkashvili We're using the latest `create-react-app`, and it uses webpack 4. So tree shaking should be enabled by default.

**Review by davidkwan95 — COMMENTED** [review]
@mctolentino it seems like there's a lot of white space changes, probably because of auto format. Should I setup pre-commit hook and that runs prettier so we don't have changes like this again in the future?

File: src/containers/sidebar.js (hunk 1 above)
Comment (denibulkashvili) [reply src/containers/sidebar.js:12]: @davidkwan95 thanks for the explanation

**Pull request merged**